account_move_promissory_note/models/account_move.py

```diff
@@ -0,0 +1,18 @@
+# © 2024 Solvos Consultoría Informática (<http://www.solvos.es>)
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
+
+from odoo import models, fields, api
+
+
+class AccountMove(models.Model):
+    _inherit = "account.move"
+
+    payment_ids = fields.Many2many('account.payment', 'account_invoice_payment_rel', 'invoice_id' , 'payment_id')
```

Comment:
Worried about it, I'm not sure if this will always link with the final payments:

https://github.com/odoo/odoo/blob/4cc086d330b73514fbc64a7fcf22d8a7a9f1b691/addons/account/models/account_payment.py#L44

and also see:

https://github.com/Comunitea/external_modules/blob/289ee428b274cd8131444ff8ac7b552cc3aeb1b3/account_due_list_payment_mode_edit/models/account_move.py#L11-L21

@lmiguens-solvos could you check?